trade-finance-manager-ui/server/controllers/helpers/get-success-banner-message.helper.getDealSuccessBannerMessage.ff-test.ts

```diff
@@ -0,0 +1,98 @@
+import { Request } from 'express';
+import { createMocks } from 'node-mocks-http';
+import { AnyObject, DEAL_SUBMISSION_TYPE, TFM_DEAL_CANCELLATION_STATUS, TfmDealCancellationWithStatus, Deal, DEAL_TYPE } from '@ukef/dtfs2-common';
+import { getScheduledCancellationBannerMessage, getDealSuccessBannerMessage } from './get-success-banner-message.helper';
+
+const getDealCancellationMock = jest.fn() as jest.Mock<Promise<Partial<TfmDealCancellationWithStatus>>>;
+
+jest.mock('../../api', () => ({
+  getDealCancellation: (params: AnyObject) => getDealCancellationMock(params),
+}));
+
+const { AIN, MIN, MIA } = DEAL_SUBMISSION_TYPE;
+
+const userToken = 'userToken';
+const dealId = 'dealId';
+const ukefDealId = 'ukefDealId';
+
+describe('getDealSuccessBannerMessage', () => {
```

Comment:
Could we document if no success message on either please

Reply:
I've gone for `expect(response).toEqual(getFlashSuccessMessage(req));` to document it defaults to that value & added undefined tests for `getFlashSuccessMessage`